### PR TITLE
[PRIMA-11880]: [Pyxis React] Aggiornare Typography stories

### PR DESCRIPTION
**Pull request by lauratrapani**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/PRIMA-11880
Overview story for typography component

### CODE REVIEW

File: src/stories/Typography/Typography.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+interface TypographyProps {
+  size: Size;
+  weight: Weight;
+  type: TypographyElement
+}
+
+export type Size = 's' | 'm' | 'l' | 'xl';
+export type Weight = 'light' | 'book' | 'bold';
+export type TypographyElement = 'text' | 'title';
```
Comment (alessiosantangelo) [inline src/stories/Typography/Typography.tsx:28]:
```suggestion
export type Type = 'text' | 'title';
```

File: src/stories/Typography/Typography.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {`${capitalize(type)} ${size.toUpperCase()}`}
+  </span>
+);
+
+export default Typography;
+
+interface TypographyProps {
+  size: Size;
+  weight: Weight;
+  type: TypographyElement
```
Comment (alessiosantangelo) [inline src/stories/Typography/Typography.tsx:23]:
```suggestion
  type: Type
```

File: src/stories/Typography/Typography.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+import classNames from 'classnames';
+import { capitalize } from '../../utils';
+
+const getClasses = (size: Size, weight: Weight, type: TypographyElement): string => classNames(
```
Comment (alessiosantangelo) [inline src/stories/Typography/Typography.tsx:5]:
```suggestion
const getClasses = (size: Size, weight: Weight, type: Type): string => classNames(
```

**Review by alessiosantangelo — CHANGES_REQUESTED** [review]

**Review by emanuelemacri — APPROVED** [review]
LGTM 🚀

**Pull request merged**